OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -133,21 +139,87 @@ static int HashList<T>(List<T> xs)
 			return hash;
 		}
 
-		List<CPos> EvalPath()
+		// NOTE TO REMOVE: This previously used mobile.ToCell. actor.Location
+		// and mobile.ToCell are exactly the same always, and using
```

Comment:
Not quite exactly the same, because `actor.Location` is a property that has additional overhead.  You could minimize that by evaluating Location once and storing it in a temporary variable.